src/rust/ensogl/src/display/shape/primitive/glsl/fragment_runner.glsl

```diff
@@ -4,9 +4,9 @@ Env   env        = Env(1);
 vec2  position   = input_local.xy ;
 Shape shape      = run(env,position);
 float alpha      = shape.color.color.raw.a;
-uint alpha_no_aa = alpha > 0.5 ? uint(1) : uint(0);
+float alpha_no_aa = alpha > 0.5 ? 1.0 : 0.0;
 
-output_id = uvec4(input_symbol_id,input_instance_id,0,alpha_no_aa);
+output_id = vec4(float(input_symbol_id),float(input_instance_id),0,alpha_no_aa);
```

Comment:
This conversion is wrong. We need bit reinterpretation here and it is needed for this task to be merged.